cmd/ingester/app/builder/builder.go

```diff
@@ -70,10 +70,11 @@ func CreateConsumer(logger *zap.Logger, metricsFactory metrics.Factory, spanWrit
 	}
 
 	consumerParams := consumer.Params{
-		InternalConsumer: saramaConsumer,
-		ProcessorFactory: *processorFactory,
-		Factory:          metricsFactory,
-		Logger:           logger,
+		InternalConsumer:  saramaConsumer,
+		ProcessorFactory:  *processorFactory,
+		Factory:           metricsFactory,
+		Logger:            logger,
+		MaxReadsPerSecond: options.MaxReadsPerSecond,
```

Comment:
ah, I think I've made a mistake when scoping this. This static config means we'd have to redeploy the ingester/indexer everytime we want to throttle vs using a dynamic config value. I think we have a way to plug in dynamic config somewhere else in this repo, let's copy that here.

Reply:
Yes. We discussed this, yet I focused on the rate limiter and hadn't thought about refactoring the config. This might be harder to do than it sounds... I have to check other places we did this.